Parity/include/VQwBPM.h

```diff
@@ -275,6 +275,7 @@ class VQwBPM : public VQwDataElement {
 /*     fAbsPos_base[1] = NULL; */
 /*     fEffectiveCharge_base = NULL; */
     fQwStriplineCalibration = 18.81; // adc counts/mm default value
+    fQwStriplineCorrection = 0.250014;
```

Comment:
DId you find out from Caryn if this depends on the physical geometry in the same way that the calibration does (i.e. does it need to be scaled by BSENfactor).

Reply:
Yeah, she says it is probably fine this way with no additional scale factor. The X^2-Y^2 term that it is modifying does already have the BSENfactor^2 in it (as well as the other relative gains, which are all set to 1.0 in our map files so far), so it seems for now that the 0.250014 is universal.